distributed/worker.py

```diff
@@ -651,7 +651,7 @@ def __init__(
                 self.nthreads, thread_name_prefix="Dask-Default-Threads"
             )
 
-        self.batched_stream = BatchedSend(interval="2ms", loop=self.loop)
```

Comment:
We might want the scheduler to return an updated interval here upon heartbeat, just like we do for the heartbeat itself

Reply:
What do you mean by that? It looks like we're similarly initializing the heartbeat interval to a high-ish value in `Worker.__init__`: https://github.com/dask/distributed/blob/92961ff9ad5958532463fc4c8ec12256dfc684b7/distributed/worker.py#L725

then in each heartbeat, the scheduler sends a [new batched send interval](https://github.com/dask/distributed/pull/5036/files#diff-0c7375c95aaa92986f8ed5460bae4e98720a14fca13556e01fd44ab6b927c24dR1002) just like it [does with the heartbeat](https://github.com/dask/distributed/pull/5036/files#diff-0c7375c95aaa92986f8ed5460bae4e98720a14fca13556e01fd44ab6b927c24dR999-R1001).

Reply:
Forget my comment. I apparently missed some code. You're doing exactly as I suggested 🤦